scout/adapter/mongo/variant_events.py

```diff
@@ -559,8 +558,56 @@ def unmark_causative(self, institute, case, user, link, variant):
 
         return updated_case
 
-    def update_manual_rank(self, institute, case, user, link, variant,
-                           manual_rank):
+    def update_cancer_tier(self, institute, case, user, link, variant, cancer_tier):
```

Comment:
This function is exactly the same as update_manual_rank. What if we use a unique function for updating manual rank and cancer tier? As the last argument of the function we could pass a tuple maybe. 

- For manual rank:('manual_rank', manual_rank)
- For cancer tier: ('cancer_tier': cancer_tier)

And then element one could be used as event verb and both of them later in find one and update. Just an idea!

Reply:
Yes, the thought crossed my mind. In event terms it's more like a verb with an object really. But I couldn't decide if it would make things simpler or more complicated in the end; especially I worried that later additions of reading some version of this from file would be complicated by storing it with manual annotation events. It may be the lazy streak speaking, but how about we keep them separate, and have a more overall look at the event structure later? The event list does keep growing in a way that suggests it could use a refactor.

Reply:
Ok, I agree!